candidate-testing.js

```diff
@@ -1,58 +1,87 @@
-const input = require('readline-sync');
+const input = require('readline-sync')
```

Comment:
Overall, great semi-colon use.  Not sure why this semi-colon got removed.  I would've left it in.  In JavaScript, semi-colons are optional, but Java requires them.